bin/ch-test

```diff
@@ -426,6 +427,10 @@ while [[ $# -gt 0 ]]; do
     --sudo)
         use_sudo=yes
         ;;
+    --lustre)
+        require_unset ch_lustre
+        export ch_lustre="$1"; shift
+        ;;
```

Comment:
1. Similar to the other PR, add `--lustre=foo` processing.
2. You don't need to `export` and can omit the double quotes.